### PR TITLE
FINAP-60/rest endpoints for pricing info

### PR DESCRIPTION
**Pull request by esuomi**
Just see the commit messages, lots of things again.

*tl;dr:* Store pricing details per service to database.

### CODE REVIEW

File: ote/src/clj/ote/services/taxiui.sql [hunk 1]
```diff
@@ -0,0 +1,2 @@
+-- name: fetch-priceinfo-for-service
+SELECT (:service)::text AS service_param;
```
Comment (esuomi) [inline ote/src/clj/ote/services/taxiui.sql:2]: Note to self: this file should be deleted

File: ote/src/cljs/taxiui/app/controller/loader.cljs [hunk 2]
```diff
@@ -0,0 +1,27 @@
+(ns taxiui.app.controller.loader
+  (:require [taxiui.app.routes :as routes]
+            [tuck.core :as tuck]))
+
+(defn- bag-counter
+  "Function for counting hits in a bag (associative data structure counting \"hits\")."
+  [m op k]
+  (let [r (op (or (get m k) 0))]
```
Comment (antti-nuortimo-solita) [inline ote/src/cljs/taxiui/app/controller/loader.cljs:8]: Where does `op` come from?

**Review by antti-nuortimo-solita — COMMENTED** [review]
Looks good to me :ship:

File: ote/src/cljs/taxiui/app/controller/loader.cljs (hunk 2 above)
Comment (esuomi) [reply ote/src/cljs/taxiui/app/controller/loader.cljs:8]: Second function parameter, the actual calls pass the operation (`inc` or `dec` in this case) that gets applied on rows 19 and 27.

**Pull request merged**